userspace/libscap/engine/gvisor/parsers.cpp

```diff
@@ -37,6 +37,8 @@ limitations under the License.
 #include <driver/syscall_compat_ppc64le.h>
 #elif __riscv
 #include <driver/syscall_compat_riscv64.h>
+#elif __loongarch64
+#include "../../driver/syscall_compat_loongarch64.h"
```

Comment:
according to our CI we need to correct this
```suggestion
#include <driver/syscall_compat_loongarch64.h>
```

Reply:
wow i didn't evne notice that others where absolute :/ 
Great catch andre!

Reply:
Oh, that's right, I forgot to change it after testing.
Thanks！